index.html

```diff
@@ -180,7 +181,7 @@ <h5>I'm always <br>interested in hearing about new projects, so if you'd like to
             name="message" id="" cols="30" rows="5" maxlength="500" required></textarea>
           </li>
           <li id="li-btn">
-            <input type="submit" 
+            <input type="submit" onclick="contactSubmit()"
```

Comment:
- [ ] [OPTIONAL] I'd advise you to separate the JS code from your HTML as this will become hard to manage when your codebase grows large. 😉